docs/cumulusci_flow.rst

```diff
@@ -0,0 +1,273 @@
+
+Once the Pull Request is merged, the feature branch is deleted.
+
+
+
+Feature Branch Flows
+^^^^^^^^^^^^^^^^^^^^
+CumulusCI facilitates working with feature branches (mainly) through two default flows:
+
+* ``dev_org``: Used to deploy the unmanaged code and all dependencies from the feature branch into a Salesforce org to create a usable development environment.
+* ``ci_feature``: Deploys the unmanaged code and all dependencies into a Salesforce org (typically a fresh scratch org) and run the Apex tests. This flow is typically run by your CI app on new commits to any feature branch.
+
+
+
+Release Branches
+----------------
+Some teams deliver large releases several times a year.
+For this type of release cadence, Salesforce.org uses a special type of branch referred to as a release branch.
+Release branches are simply a feature branch named with a number.
+These long-lived branches are created off of the ``main`` branch, serve as the target branch for all
+features associated with that release and are eventually merged back to the ``main`` branch when a release occurs.
+To be able to clearly track what work is associated with a specific release, release branches adhere to the following:
+
+* They are the parent branches of ALL feature work associated with a release. Put another way; all feature branches use the parent-child naming convention with its target release branch.
+* Use a strict naming format: ``feature/release_num`` where ``release_num`` is a valid integer.
+
+Using ``feature/`` branch prefix for the release branch names allow those branches to stay in sync with our main branch (they are just another feature branch to CumulusCI).
+The release number immediately after the ``feature/`` prefix allows CumulusCI to detect and merge changes from one release branch to other future release branches.
+See `Release to (Future) Release Merges`_ for more information.
+
+An example of a release branch with two items of work associated with it could look like this:
+
+* ``feature/001``
+* ``feature/001__feature1``
+* ``feature/001__feature2``
+
+
+
+Branch Configuration
+--------------------
+The name of the main (default) branch, as well as the branch prefixes are configurable in your projects ``cumulusci.yml`` file.
+The following shows the default values that CumulusCI comes with:
+
+.. code-block:: yaml
+
+   project:
+      git:
+         default_branch: main
+         prefix_feature: feature/
+         prefix_beta: beta/
+         prefix_release: release/
+
+These values can be changed to match naming conventions used by your own project.
+
+
+
+.. _auto-merged:
+
+Auto Merging
+------------
+CumulusCI helps to keep large diffs and merge conflicts from being the norm. CumulusCI's auto-merge functionality helps teams:
```

Comment:
```suggestion
CumulusCI Flow helps to keep large diffs and merge conflicts from being the norm. CumulusCI's auto-merge functionality helps teams:
```

This is also a key benefit we could articulate in the introduction, and then reiterate here.